root/documentation/compara.conf

```diff
@@ -254,9 +254,25 @@
       </aligned>
       <sequence>
         type=Enum(none, cdna, protein)
-        description=The type of sequence to bring back. Setting it to none results in no sequence being returned
+        description=The type of sequence to bring back. Setting it to none results in no sequence being returned. note: dependant on the aligned option. if aligned option is set to 1, this is able to override the sequence=none option. sequence=cdna returns an error unless the uniprot members are filtered out by setting member_source=ensembl
```

Comment:
Maybe word this as "Note: dependant on the setting for "aligned." If aligned=1, this will override sequence=none"